templates/CRM/Admin/Page/ScheduleReminders.tpl

```diff
@@ -8,15 +8,15 @@
  +--------------------------------------------------------------------+
 *}
 {* this template is for configuring Scheduled Reminders *}
-{if $setTab eq 1}
-  {if $component eq 'event'}
+{if !empty($setTab) and $setTab eq 1}
+  {if !empty($component) and $component eq 'event'}
```

Comment:
ok - we actually have been switching to && in the templates for consistency - but non blocking